src/Autocomplete.js

```diff
@@ -58,13 +58,13 @@ class Autocomplete extends AutocompleteBase {
 	}
 
 	/**
-	 * Focuses the option at the given index.
+	 * Actives an option of the suggestion list by inform an index.
 	 * @param {number} index
 	 * @protected
 	 */
-	focusIndex_(index) {
+	activeListItem_(index) {
 		let option = this.currentList_[index];
-		this.focusedIndex_ = index;
+		this.activedIndex_ = index;
```

Comment:
The correct in English would be either `activatedIndex_` or `activeIndex_`, not `activedIndex_`, can you do this small fix? Same for `activeListItem_`, it should be `activateListItem_`.